src/software/sensor_fusion/filter/ball_filter.h

```diff
@@ -43,11 +43,13 @@ class BallFilter
     static constexpr double MAX_BUFFER_SIZE_VELOCITY_MAGNITUDE = 4.0;
     // The extra amount beyond the ball's max speed that we treat ball detections as valid
     static constexpr double MAX_ACCEPTABLE_BALL_SPEED_BUFFER = 2.0;
+    // the speed after which we start to experience rolling friction
+    static constexpr double BALL_MIN_SPEED_FOR_ROLLING_ACCELERATION = 0.2;
```

Comment:
Why is this necessary? Why can't we always apply acceleration?

Reply:
friction will only be applied if the ball is in motion, but to account for vision not being completely accurate theres a +- 0.2